src/redisraft.c

```diff
@@ -1297,6 +1297,111 @@ static int cmdRaftRandom(RedisModuleCtx *ctx,
     return REDISMODULE_OK;
 }
 
+static int cmdRaftScan(RedisModuleCtx *ctx,
```

Comment:
currently the api is

raft.scan <cursor> slots [slot-range]

perhaps it can be

raft.scan [slot-range] <cursor>